doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -0,0 +1,703 @@
+```
+
+If you have set up the observability components (see the Installation guide), and remote write to a Thanos instance, then you should be able to access the Grafana instance and see your deployed gateway and policies in the `platform` engineer dashboard.
+
+## Platform Engineer review
+
+We have now established an external gateway, secured it with TLS, and protected all endpoints with a default `DENY ALL` AuthPolicy, alongside a restrictive default `RateLimitPolicy`. We have also configured a `ManagedZone` and a `DNSPolicy` to direct traffic to the gateway via the specified listener hosts. Our gateway is now ready to begin receiving traffic.
+
+By creating an `HTTPRoute` for our listeners, we will activate the `DNSPolicy`, `AuthPolicy`, and `RateLimitPolicy`, populating DNS records and configuring auth and rate limiting to safeguard requests to that endpoint.
+
+To verify the setup, we can deploy a simple application and connect it to our gateway:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${gatewayNS}
+```
+
+Add an `HTTPRoute` to expose this application:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: test
+  namespace: ${gatewayNS}
+spec:
+  parentRefs:
+  - name: ${gatewayName}
+    namespace: ${gatewayNS}
+  hostnames:
+  - "test.${rootDomain}"
+  rules:
+  - backendRefs:
+    - name: toystore
+      port: 80
+EOF
+```
+
+Check our gateway policies are enforced:
+
+```bash
+kubectl get dnspolicy ${gatewayName}-dnspolicy -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get authpolicy ${gatewayName}-auth -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get ratelimitpolicy ${gatewayName}-rlp -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+```
+
+**Note:** `TLSPolicy` is currently missing an enforced condition. https://github.com/Kuadrant/kuadrant-operator/issues/572. However, looking at the gateway status we can see it is affected:
+
+```bash
+kubectl get gateway -n ${gatewayNS} ${gatewayName} -n ${gatewayNS} -o=jsonpath='{.status.conditions[*].message}'
+```
+
+### Test connectivity and deny all auth 
+
+We are using `curl` to hit our endpoint. As we are using LetsEncrypt staging in this example, we pass the `-k` flag:
+
+```bash
+curl -k -w "%{http_code}" https://$(kubectl get httproute test -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
+```
+
+We should see a `403` response. With our gateway and policies in place, we can now allow other teams to use the gateway:
+
+```bash
+kubectl patch gateway ${gatewayName} -n ${gatewayNS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
+```
+
```

Comment:
```suggestion
### Test connectivity and deny all auth 

You can use `curl` to hit your endpoint. Because this example uses Let's Encrypt staging, you can pass the `-k` flag:

```bash
curl -k -w "%{http_code}" https://$(kubectl get httproute test -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
```

You should see a `403` response. With your Gateway and policies in place, you can now allow other teams to use the Gateway:

```bash
kubectl patch gateway ${gatewayName} -n ${gatewayNS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
```

```